.travis.yml

```diff
@@ -0,0 +1,15 @@
+---
+language: python
+python:
+  - "2.7"
+  - "3.6"
+install:
+  - gem install mdl
+  - pip install rstcheck
+  - pip install sphinx
+script:
+  # Linting
+  # https://github.com/markdownlint/markdownlint
+  - mdl -s relaxed README.md
+  # https://github.com/myint/rstcheck
+  - rstcheck --report warning source/*.rst
```

Comment:
not brave enough to put `--report info` :)

Reply:
The fact is that it was complaining about missing usage of references that where in fact existing in different files. :)